### ISSUE TITLE
Cannot add credentials / API Key in n8n

### ISSUE THREAD

**Issue by ngrrd** [go-vikunja/n8n-vikunja-nodes#57]
Hey there!

I'm not very familiar with Github processes, so I hope I had to create a new issue rather than comment on this one, since it's been closed : https://github.com/go-vikunja/n8n-vikunja-nodes/issues/52

I've just installed Vikunja today, version 0.23.0. At the same time, I installed the community node for n8n, also in the latest version, 0.1.3. This version seems to have fixed the problem for Niskeletor but not for me. (the difference being that this is the only version I've installed directly, I've never updated the node since I just installed it)
![image](https://github.com/go-vikunja/n8n-vikunja-nodes/assets/76825943/bb391d8e-e4f3-48da-9437-2057843f3d26)

n8n is version 1.41.1, installed on Docker
![image](https://github.com/go-vikunja/n8n-vikunja-nodes/assets/76825943/a994dbd9-95b3-464d-910e-c2e374ba44e6)

The problem seems to be identical to Niskeletor, I can create a new trigger but I can't add credentials :
![image](https://github.com/go-vikunja/n8n-vikunja-nodes/assets/76825943/106688bf-e5e9-40cb-8bd0-1605d2992e9b)

![image](https://github.com/go-vikunja/n8n-vikunja-nodes/assets/76825943/32548564-69d0-43d4-8b74-2ab013c0c7be)

I tried restarting n8n container without success.

Thanks for your help.
I can't wait to test Vikunja, which looks very promising.

Edit : I've been offered n8n update 1.42.1, I've done it, I've also uninstalled and reinstalled n8n-nodes-vikunja but I still can't add credentials

**Comment by kolaente** [comment]
Hi!

Is there anything in n8n's logs?

**Comment by jochem3k** [comment]
Just chiming in to say I've been experiencing the exact same issue. Nothing in the n8n logs either.

**Comment by weits** [comment]
I have the same problem too.

**Comment by ngrrd** [comment]
Hello @kolaente thanks for your answer. I won't be able to check any logs as I completely broke my host machine after an update and decided to reinstall everything. :-)
BUT : before that, I don't know why because I didn't do anything more, but I was able to add the n8n credentials.
As I said I had previously done the n8n 1.42.1 update and it didn't change anything. But later I tried adding Credentials again and found that it worked. I didn't go through the “Credentials” tab in n8n but directly by creating a new Vikunja trigger, and surprise, I was able to add Credentials there.
It's not all very precise, but maybe @jochem3k and @weits will add more to this ?
To be clear, when this didn't work, I tried adding Credentials via the Credentials page as well as via a new Vikunja trigger and none of it worked. And then later it started to work, but I don't know why.
I'll be reinstalling n8n and Vikunja soon, so I can see if I have the same problem.
